build/async.js

```diff
@@ -0,0 +1,1225 @@
+'use strict';
```

Comment:
Do we want this file to be called `build/async` or `build/index`. Also on publish are we going to move the `/build` files to `/` so people can do `import each from 'async/each`?